src/bluesearch/entrypoint/database/parse.py

```diff
@@ -14,49 +32,72 @@
 def get_parser() -> argparse.ArgumentParser:
     """Create a parser."""
     parser = argparse.ArgumentParser(
-        description="Parse article.",
+        description="Parse one or several articles.",
     )
     parser.add_argument(
         "article_type",
         type=str,
         choices=("cord19-json", "pmc-xml"),
-        help="""Article source type.""",
+        help="""
+        Article format. If parsing several articles, all articles
+        must have same format.
+        """,
     )
     parser.add_argument(
         "input_path",
-        type=str,
-        help="""Path to the file/directory to be parsed.""",
+        type=Path,
+        help="""
+        Path to a file or directory. If a directory, all articles
+        inside the directory will be parsed.
+        """,
     )
     parser.add_argument(
         "output_path",
-        type=str,
-        help="""Path where the parsed article is saved.""",
+        type=Path,
+        help="""
+        Path to a directory where parsed article(s) will be saved.
+        If it does not exist yet, a directory with this path is created.
+        """,
     )
     return parser
 
 
 def run(
     *,
     article_type: str,
-    input_path: str,
-    output_path: str,
+    input_path: Path,
+    output_path: Path,
 ) -> None:
-    """Parse an article.
+    """Parse one or several articles.
 
     Parameter description and potential defaults are documented inside of the
     `get_parser` function.
     """
-    parser_inst: ArticleParser
-    if article_type == "cord19-json":
-        with open(input_path) as f_input:
-            parser_inst = CORD19ArticleParser(json.load(f_input))
-    elif article_type == "pmc-xml":
-        parser_inst = PubmedXMLParser(input_path)
-
+    inputs: Iterable[Path]
+    if input_path.is_file():
+        inputs = [input_path]
+    elif input_path.is_dir():
+        inputs = input_path.glob("*")
```

Comment:
Does the `glob(*)` have some sorting logic?

Reply:
As you correctly said, `Path.glob()` does not guarantee any specific order.
Fixed in ae81762 to guarantee reproducibility.